rr/ai/ai_manager.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+import gi  # nopep8
+gi.require_version('Gst', '1.0')  # nopep8
+gi.require_version('GLib', '2.0')  # nopep8
+from gi.repository import Gst as gst  # nopep8
+from gi.repository import GLib  # nopep8
+
+from TI.postprocess import PostProcessDetection
+from TI.preprocess import PreProcessDetection
+from TI.runtimes import *
+
+
+class AIManagerError(RuntimeError):
+    pass
+
+
+class AIManager():
+    """
+    Class that handles the AI modules
+
+    Attributes
+    ----------
+    preprocess_obj : PreProcessDetection object
+        The PreProcessDetection object
+
+    preprocess_obj : PreProcessDetection object
+        The PreProcessDetection object
+
+    Methods
+    -------
+    preprocess_detection(media : media input)
```

Comment:
Please rename media to image to avoid confusion with the media objects. (please replace all locations)